docs/topics/run-code-snippets.md

```diff
@@ -55,30 +55,125 @@ To rebuild the module automatically before each run of a scratch or a worksheet,
 
 ![Scratch select module](scratch-select-module.png){width=700}
 
-### Running as REPL 
+### Run as REPL 
 
-To evaluate each particular expression in a scratch or a worksheet, run it with **Use REPL** selected. The code will be 
-executed the same way as in [REPL](#repl): the code lines will run sequentially, providing results of each call. 
+To evaluate each particular expression in a scratch or a worksheet, run it with **Use REPL** selected. The code lines
+will run sequentially, providing results of each call. 
 You can later refer to the results by the names `res*` shown in the corresponding lines.
 
 ![Scratch REPL](scratch-repl.png){width=700}
 
-## REPL
+## Browser: Kotlin Playground
 
-_REPL_ (_Read-Eval-Print-Loop_) is a tool for running Kotlin code interactively. REPL lets you evaluate expressions and 
-code chunks without creating projects or even functions if you don't need them. 
+[Kotlin Playground](https://play.kotlinlang.org/) is an online application that allows writing, running, and sharing
+Kotlin code in the browser.
 
-To run REPL in IntelliJ IDEA, open **Tools** | **Kotlin** | **Kotlin REPL**.
+### Write and edit code
 
-To run REPL in the OS command line, open `/bin/kotlinc-jvm` from the directory of standalone Kotlin compiler.
+In the Playground editor area, you can write code as you would write in a source file:
+* place your own classes, functions, and top-level declarations in an arbitrary order
+* write the executable part in the body of the `main` function.
 
-The REPL command line interface will open. You can enter any valid Kotlin code and see the result. The results are printed as 
-variables with auto-generated names like `res*`. You can later use such variables in the code you run in REPL.
+Like in usual Kotlin projects, the `main` function in the Playground can have the `args` parameter or no parameters at all.
+To pass program arguments upon execution, write them in the **Program arguments** field.
 
-![Run REPL](repl-run.png){width=700}
+![Playground: code completion](playground-completion.png){width=700}
 
-REPL supports multiline input as well. The result of the multiline input is the value of its last expression. 
+The Playground highlights the code and shows code completion options as you type. It automatically imports declarations
+from the standard library and [`kotlinx.coroutines`](coroutines-overview.md).
 
-![REPL multi-line run](repl-multi-line.png){width=700}
+### Choose execution environment
 
+The Playground provides means for customizing the execution environment:
+* multiple Kotlin versions, including available previews of a future version
```

Comment:
Link to the EAP? 
Or it could just distract user? 🤔

Reply:
Yeah, why not. Thanks for the idea